hibp.js

```diff
@@ -117,7 +134,7 @@ const HIBP = {
     // ]
     for (const breachedAccount of response.body) {
       if (sha1.toUpperCase() === sha1Prefix + breachedAccount.hashSuffix) {
-        foundBreaches = allBreaches.filter(breach => breachedAccount.websites.includes(breach.Name));
+        foundBreaches = allBreaches.filter(breach => breachedAccount.websites.includes(breach.Title));
```

Comment:
This feels like a curious change... Why is foundBreaches using `.Title`s now instead of `.Name`?